### PR TITLE
Update authentication documentation.

### PR DESCRIPTION
**Pull request by beefo**
- Federation/numbered servers is no longer relevant.
- Update the authenticate code example to show how to use path.
- Shuffle sections around a little.
- Remove HTTP GET example. We want people to use POST.

### CODE REVIEW

File: src/software/guides/concepts.md [hunk 1]
```diff
@@ -473,6 +467,112 @@ statusDatas[i].device = deviceLookup[statusDatas[i].device.id];
...
+### Examples
+
+A simple [example]({{site.baseurl}}/software/api/runner.html#sample:get-lightweight-device-response) of this can be illustrated by using the property selector with `Device`. The `Device` object can have many properties which may not be useful to all use-cases. For example, if I have an add-in to display a list of 500 devices by name. We only want our `Device` objects to have the properties `Name` and `Id`, so we set our `PropertySelector` object like so:
+
+#### Javascript
+
+##### Request
+
+```javascript
+api.call("Get", {
```
Comment (ravneet-s) [inline src/software/guides/concepts.md:492]: Currently a simple Get call without search does not work with propertyselector for several entities including device. I have a case to fix this. Is it okay to have this example here before the fix?

File: src/software/guides/concepts.md [hunk 2]
```diff
@@ -473,6 +467,112 @@ statusDatas[i].device = deviceLookup[statusDatas[i].device.id];
...
+  });
+```
+
+## List of Supported Entities
+
+Below is a list of entities that are support the PropertySelector functionality.
+
+| **Entity** | **Supported in Release** | **Notes**
+| --- | --- | -- |
+| [Device]({{site.baseurl}}/software/api/reference/#Device) | 8.0 | The following properties are not supported: `deviceFlags`, `isAuxInverted`, `deviceType`, `productId`, `autogroups`, `auxWarningSpeed`, `enableAuxWarning`
```
Comment (ravneet-s) [inline src/software/guides/concepts.md:560]: Same here. Should we specify it only works with search and then update later once the Get is fixed?

**Review by ravneet-s — COMMENTED** [review]
There were some minor spelling checks that I have updated already.

File: src/software/guides/concepts.md (hunk 1 above)
Comment (beefo) [reply src/software/guides/concepts.md:492]: @ravneet-s this example looks to work as expected as tested. I get back device objects with only id and name populated.

**Pull request merged**